taipy/gui/_renderers/factory.py

```diff
@@ -372,6 +372,9 @@ class _Factory:
                 ("show_value", PropertyType.boolean, True),
                 ("format", PropertyType.string),
                 ("delta_format", PropertyType.string),
+                ("template", PropertyType.dict),
+                ("template[dark]", PropertyType.dict),
+                ("template[light]", PropertyType.dict),
```

Comment:
Please create an issue for document that thing.
True for chart as well.
Thanks!

Reply:
Added:
https://github.com/Avaiga/taipy/issues/1368
https://github.com/Avaiga/taipy/issues/1369